src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -36,20 +93,46 @@ public void RenewalWithOutstandingHoldRequestDoesNotChangeItemStatus()
       .withPickupServicePointId(pickupServicePointId)
       .by(usersFixture.charlotte()));
 
-    loansFixture.renewLoan(smallAngryPlanet, rebecca);
+    Response response = renewByIdClient.attemptCreate(
+      new RenewByIdRequestBuilder()
+        .forItem(smallAngryPlanet)
+        .forUser(rebecca));
 
-    Response changedItem = itemsClient.getById(smallAngryPlanet.getId());
+    assertThat(response.getStatusCode(), is(422));
 
-    assertThat(changedItem.getJson().getJsonObject("status").getString("name"),
-      is("Checked out"));
+    String message = response.getJson()
+      .getJsonArray("errors")
+      .getJsonObject(0)
+      .getString("message");
+
+    assertThat(message, is("Items cannot be renewed when there is an active recall request"));
```

Comment:
There are custom matchers to help check for validation error responses, for [example](https://github.com/folio-org/mod-circulation/blob/931d6ac1cafa9c0511199a71b6292014d3fda5fc/src/test/java/api/requests/scenarios/SingleOpenHoldShelfRequestTests.java#L113)

Reply:
) Thanks

Reply:
Done